pkg/aggregator/aggregator.go

```diff
@@ -653,6 +718,17 @@ func (agg *BufferedAggregator) Flush(start time.Time, waitForSerializer bool) {
 	agg.flushSeriesAndSketches(start, waitForSerializer)
 	agg.flushServiceChecks(start, waitForSerializer)
 	agg.flushEvents(start, waitForSerializer)
+	// Event Platform Events are forwarded continuously instead of being buffered/aggregated in memory like the other
+	// types of telemetry. They are buffered in memory only when flushing is disabled to allow the agent check command
+	// to read them out. Even though they are not buffered in memory under normal operation we still trigger a flush
+	// here to prevent endless growth just in case the aggregator is being misused somehow.
+	agg.GetEventPlatformEvents(false)
+}
+
+func (agg *BufferedAggregator) logPostFlushMessages() {
+	if aggregatorEventPlatformLatestError != nil {
+		log.Warnf("Failed to process some event platform events. latest-error='%s' eventCounts=%s errorCounts=%s", aggregatorEventPlatformLatestError, aggregatorEventPlatformEvents.String(), aggregatorEventPlatformEventsErrors.String())
+	}
```

Comment:
What the point of this ? Why not simply print every error or a percentage ? We would also print the same error message multiple times, unless I'm missing the reset.

Reply:
See my comment here https://github.com/DataDog/datadog-agent/pull/7735#discussion_r612702180